content/company-info-and-process/communication/team_chat.md

```diff
@@ -14,7 +14,7 @@ Slack is not a [source of truth](https://sourcegraph.com/github.com/sourcegraph/
 
 ### Naming conventions
 
-To help with navigation and discoverability, follow these naming conventions for channels. If you see find a channel that does not follow these, request to rename it in [#ask-it-tech-ops](https://sourcegraph.slack.com/archives/C01CSS3TC75)
```

Comment:
The link to the Slack channel is unnecessary in the handbook as channel names are automatically detected and formatted as links by the handbook build tool:

<img width="262" alt="image" src="https://github.com/sourcegraph/handbook/assets/8942601/25369a3c-669e-46a7-951d-697d9345d4ec">

The Slack channel visual formatting is actually slightly different from normal link formatting, too, so removing the link provides a more consistent and distinct appearance for Slack channel references.